scripts/artifacts-building/containers/test-built-container-idempotency-test.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: Dynamic inclusion of container
+  hosts: localhost
+  connection: local
+  gather_facts: no
+  vars_files:
+    - container-vars.yml
+  vars:
+    image_name: "{{ lookup('env', 'CONTAINER_TYPE') | default('keystone', True) }}"
+  tasks:
+    - name: prepare group for integrity checking
+      add_host:
+        name: "/tmp/{{ image_name }}/"
+        groups: "lxc_container_artifact"
+        container_name: "LXC_NAME"
+        physical_host: "localhost"
+      changed_when: false
+
+- name: Check integrity
+  hosts: lxc_container_artifact
+  connection: chroot
+  # Gether facts is necessary
+  vars_files:
+    - container-vars.yml
+  vars:
+    image_name: "{{ lookup('env', 'CONTAINER_TYPE') | default('keystone', True) }}"
+  tasks:
+    - name: Fetch get-pip for comparison
+      get_url:
+        url: "{{ rpco_mirror_base_url }}/os-releases/{{ rpc_release }}/get-pip.py"
+        dest: /opt/get-pip.py
```

Comment:
Well if I fetch to a different location, it's gonna be marked as changed. Here everything is done naturally.